Source/WebCore/Modules/webxr/XRGPUBinding.idl

```diff
@@ -26,6 +26,7 @@
 // https://github.com/immersive-web/WebXR-WebGPU-Binding/blob/main/explainer.md
 [
     Conditional=WEBXR_LAYERS,
+    Conditional=WEBGPU,
```

Comment:
Is this defined anywhere? 

In the other idl files we are using `EnabledBySetting=WebGPUEnabled`

Reply:
It's true that there is no default value in `PlatformEnable.h`, we might consider adding it there too, but it should be available for builds.

Reply:
Adding to PlatformEnable.h would be fine :)